testing/smoke-ide-test/build.gradle.kts

```diff
@@ -90,10 +91,11 @@ dependencies {
     smokeIdeTestDistributionRuntimeOnly(projects.distributionsFull) {
         because("Tests starts an IDE with using current Gradle distribution")
     }
+    smokeIdeTestImplementation(libs.gradleIdeStarterScenarios)
     smokeIdeTestImplementation(testFixtures(projects.core))
 }
 
-integTest.testJvmXmx = "1g"
+integTest.testJvmXmx = "5g"
```

Comment:
Incremental sync requires some memory, so I've increased test process's XMX for that.
@gradle/bt-developer-productivity Is it legal?